app/assets/javascripts/analytics/_virtualPageViews.js

```diff
@@ -16,15 +16,24 @@
 
   };
 
-  VirtualPageViews.prototype.sendVirtualPageView = function( url, name ) {
+  VirtualPageViews.prototype.sendVirtualPageView = function( url, name, dimensions ) {
 
-    if ( GOVUK.analytics  && url ) {
-      var urlList = url.split("?");
-      urlList[0]  = urlList[0] + "/vpv";
-      url         = urlList.join("?");
-      GOVUK.analytics.trackPageview(url, name);
+    if ( !GOVUK.analytics || !url ) return false;
```

Comment:
I like how you changed it to be a guard statement :)

Reply:
:)